app/controllers/sessions_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
 
     if @user && (@user.mfa_enabled? || @user.webauthn_credentials.any?)
       setup_webauthn_authentication(form_url: webauthn_create_session_path, session_options: { "user" => @user.id })
```

Comment:
`session_options: { "user" => @user.id }`

could we use `session[:mfa_user]` instead now in webauthn_create?

(Can punt this to later)

Reply:
Yeah I think we can rely on `session[:mfa_user]` instead. But will save that for a separate issue.